### ISSUE TITLE
Home Page Cards

### ISSUE THREAD

**Issue by sumana2001** [betaoverflow/project-orsi#6]
![image](https://user-images.githubusercontent.com/63084088/115954939-555b3580-a511-11eb-80ac-2ddfb691523a.png)
Add cards similar to this below the hero section (above about section) in the home page.
It'll be great if it has similar shadow effect or you can add something else of your own as well. Make sure the cards are responsive.
Please add a screenshot with the pull request.

**Comment by tanishghosh** [comment]
Hi ! I would like to work on this issue.

**Comment by sumana2001** [comment]
Sure go ahead😄

**Comment by rushiv7** [comment]
Hi! Sorry I was working on this task without asking you guys. I am new to react so I thought it will be hard for me but I have tried to design it as specified above. 
I have added svg icons which I found suitable and downloaded from [react svg icons](https://reactsvgicons.com/).
can I raise the pull request?

![Screenshot 2021-04-25 172649](https://user-images.githubusercontent.com/39614961/115992799-27e3ba00-a5ed-11eb-8fed-e4409cac51ce.png)
